typescript-arrays/main.ts

```diff
@@ -0,0 +1,27 @@
+const colors: string[] = ['red', 'white', 'blue'];
+const firstColor: string = colors[0];
+const secondColor: string = colors[1];
+const thirdColor: string = colors[2];
+console.log('value of colors[0]:', firstColor);
+console.log('value of colors[1]:', secondColor);
+console.log('value of colors[2]:', thirdColor);
+
+const AmericaColors: string = `America is ${firstColor} ${secondColor} ${thirdColor}`;
+console.log(AmericaColors);
+
+colors[2] = 'green';
+const MexicoColors: string = `Mexico is ${firstColor} ${secondColor} ${thirdColor}`;
+console.log(MexicoColors);
+
+console.log('value of colors:', colors);
+console.log('typeof colors:', typeof colors);
+
+const students: string[] = ['charlie', 'billy', 'ken', 'ben'];
+const numberOfStudents: number = students.length;
+console.log('There are four students:', numberOfStudents);
+const lastIndex: number = numberOfStudents - 1;
+console.log('the last student in array is', lastIndex);
```

Comment:
You don't need this line because we don't need the lastIndex to be logged. It also makes it look like "the last student in the array is 3" whereas your console log on line 26 is the correct value of "the last student in the array is ben"

tl;dr - you can remove this line